docker-compose.yml

```diff
@@ -28,16 +24,12 @@ services:
     image: openpodcast/spotify-connector:main
     env_file:
       - .env.spotify
-    environment:
-      - CRON_SCHEDULE=0 10 * * *
```

Comment:
I realize that this is the default value, but maybe we should keep this line to show how the schedule could be overwritten?

Reply:
exactly, that was my idea.